frontend/integration-tests/views/devconsole-view/template.view.ts

```diff
@@ -0,0 +1,64 @@
+// import * as _ from 'lodash';
+import { $, $$, browser, by, ExpectedConditions as until } from 'protractor';
+
+import { appHost, testName, waitForNone } from '../../protractor.conf';
+import { resourceTitle } from '../../views/crud.view';
+
+
+export const reloadBtn = $('#reload-object');
+
+//element finder with class name
+export const untilLoadingBoxLoaded = until.presenceOf($('.loading-box__loaded'));
+export const untilNoLoadersPresent = waitForNone($$('.co-m-loader'));
```

Comment:
We should be encouraging the use of `data-test-id` for new tests in place of class selectors.